### PR TITLE
Show default partners on home page

### LINKED ISSUE #71: Homepage - Replace the partners logos

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Show default partners on home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the partner list and compare to [design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5d9b4b74e301820685aebb72)

## Issue / Card

Fixes #71

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -228,71 +228,78 @@ <h2 class="p-heading-highlight--left__item p-heading--one">Partners make our pla
...
-          {% for partner in partner_groups[partner_group] %}
-          <li class="p-inline-images__item">
-            <img class="p-inline-images__logo" data-src="{{ partner['logo'] }}" alt="">
-          </li>
-          {% endfor %}
-        </ul>
-        {% endfor %}
-      </div>
+    <div class="u-fixed-width">
+      <ul class="p-inline-images" data-partner="default">
```
Comment (sowasred2012) [inline templates/index.html]: is this data attribute still needed?

**Comment by waynecrosby** [comment]
@solazio - Just a couple of things:
1. Can we reduce the size of Dell and increase GoogleCS, or is it set to max width & height? If so don't worry
2. Can you know China Telecom down onto the bottom line so that Yahoo Japan is not on it's own

File: templates/index.html (hunk 1 above)
Comment (solazio) [reply templates/index.html]: not really, will remove it

**Comment by solazio** [comment]
> @solazio - Just a couple of things:
> 
> 1. Can we reduce the size of Dell and increase GoogleCS, or is it set to max width & height? If so don't worry
> 2. Can you know China Telecom down onto the bottom line so that Yahoo Japan is not on it's own

@waynecrosby I reduced the size of Dell logo and made Google bigger. The logos are positioned responsive depending on your screen size. Please review again.

**Pull request merged**